src/client/pages/Checkout/shared/types.ts

```diff
@@ -1,5 +1,16 @@
 import { ExtraBuildingInput } from 'data/graphql'
-import { InsuranceType } from 'utils/hooks/useSelectedInsuranceTypes'
+
+// TODO: we should get this from giraffe schema
```

Comment:
This was the only thing left in `useSelectedInsuranceTypes.ts` and it didn't make sense to keep that unused hook file just for some type definitions so I moved them here. Not sure if there's a better place to put them?